ppmclibs/tinycv.pm

```diff
@@ -22,6 +22,7 @@ bootstrap tinycv $VERSION;
 package tinycv::Image;
 
 use Mojo::Base -strict, -signatures;
+use Text::Levenshtein qw(distance);
```

Comment:
If you introduce a new dependency, you need to update dependencies.yaml and regenerate the spec/container files.

Reply:
Thanks. I will make sure to do this. However, regarding this particular dependency we might need to get it in the Tumbleweed repos before we can merge this PR.